site/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "hexo": {
-    "version": "3.2.2"
+    "version": ""
```

Comment:
This is a `hexo` set parameter (which I believe it uses to help in automated migrations), I'm not sure setting it to blank will do anything other than resulting it getting re-set the next time `hexo` is run, but I'm willing to try it!

Reply:
Follow-up, it didn't work out: https://github.com/meteor/blaze/pull/284/commits/0d1a4e3a46a941a54f4f1fdb6542cf9ef0f46181.  No worries!